src/components/Nav.tsx

```diff
@@ -10,7 +10,7 @@ import MenuItem from "@mui/material/MenuItem"
 import Toolbar from "@mui/material/Toolbar"
 import Typography from "@mui/material/Typography"
 import { makeStyles } from "@mui/styles"
-import i18n from "i18next"
+import { default as i18n } from 'i18next'
```

Comment:
i don't know why this work.
I followed this guide: https://github.com/i18next/i18next/issues/1692

Reply:
Eslint was so strict with this rule: [no-named-as-default-member](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/no-named-as-default-member.md) 

The module has `i18n` as default export and other export properties like `use`, `changeLanguage`. According to eslint, we need separate them as <`import i18n, {use, changeLanguage} from "i18next"`> or in this case, I just import everything from the module: <`import * as i18n from "i18next"`>